Apps/Contoso.WPF.Puppet/MainWindow.xaml.cs

```diff
@@ -95,6 +95,38 @@ private void TrackEvent_Click(object sender, RoutedEventArgs e)
             Analytics.TrackEvent(name, propertiesDictionary);
         }
 
+        private void CountryCodeEnabled_Checked(object sender, RoutedEventArgs e)
+        {
+            if (cbEnableCountryCode.IsChecked.HasValue)
```

Comment:
I think this should be refactored to make it more readable

Reply:
Not resolved yet, reactivating the comment.
We can use guardian pattern here to reduce nesting